benchmark/script.py

```diff
@@ -244,6 +241,22 @@ def main(argv, environ, react=react):
     except UsageError as e:
         usage(options, e.args[0])
 
+    if options['log-file'] is not None:
+        try:
+            log_file = open(options['log-file'], 'a')
```

Comment:
I guess that different people may have different expectations about the behavior. I do not have a strong opinion on this issue, but I followed an example set in `run-acceptance-tests` (`admin/acceptance.py`) where a log file is opened in `"a"` mode.  So, I've got used to that behavior.
I guess we could add another option which would tell whether to append to a log or to overwrite it, but that feels like an overkill.